apps/src/StudioApp.js

```diff
@@ -733,61 +728,30 @@ StudioApp.prototype.loadBlocks = function(blocksXml) {
 *    block position.
 */
 StudioApp.prototype.arrangeBlockPosition = function(startBlocks, arrangement) {
```

Comment:
Would be great if there were a comment above this method explaining what it does/doesn't do now.